lib/yext/api/utils/configuration.rb

```diff
@@ -55,6 +55,23 @@ module Utils
       #
       # sandbox
       #   Boolean that indicates if the gem should use the production or sandbox URL.
+      #
```

Comment:
Readme should be updated with call back details even if it points user to this doc.